.github/workflows/main.yml

```diff
@@ -37,6 +37,7 @@ jobs:
 
   integration-tests:
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
was there a reason why we had it not continue when we encounter a failure before? I assume it was to make sure it doesn't take too much time when it does fail. On that note, why did we decide to change?

Reply:
@dsawali I just cherry-picked the commit from Dan to fix the integration tests run that were canceling by themselves.